lib/component.js

```diff
@@ -32,11 +32,19 @@
   if (typeof define === 'function' && define.amd) {
     define(['react', 'backbone', 'underscore'], factory);
   } else if (typeof module !== 'undefined' && module.exports) {
-    module.exports = factory(require('react'), require('backbone'), require('underscore'));
+    var react = null;
+    var isReactNative = false;
+    try {
+      react = require('react');
+    } catch (error) {
+      react = require('React'); // React Native
```

Comment:
You can find the source file in [here](https://github.com/magalhas/react-native-to-do-example/blob/master/index.ios.js)